tests/functional/comments-auth.js

```diff
@@ -41,35 +41,6 @@ registerSuite("Comments (auth)", {
       );
     },
 
-    "Posting a comment"() {
-      var originalCommentsLength;
-      var allCommentsLength;
-      return (
-        FunctionalHelpers.openPage(this, url("issues/100"), ".js-image-upload")
-          .findAllByCssSelector(".js-Issue-comment")
-          .then((elms) => {
-            originalCommentsLength = elms.length;
-          })
```

Comment:
not very familiar with this part of the app, but wonder why this test is no longer needed?

Reply:
Good question! It's probably still needed, but the way this passed before was that it never really posted a new comment, but it added a new comment to the DOM. So we were just testing that that succeeded. In this current PR, new comments are only added to the UI after we get a 201 back from GitHub. It might be possible to mock this out... I'll have to think (and maybe file a follow-up bug).

Reply:
Actually, we don't need to do much to get this working. I'll push another commit. 😄